developer/community-resources.md

```diff
@@ -0,0 +1,140 @@
+# Curated list of community-provided resources for Reaction
+
+This page is here to guide you through the Reaction ecosystem. As we've stated before, we'd not be here without our very engaged community. Lots of people are continuing to contribute to core development, but also supplying other members of the community with valuable plugins, tutorials or learning material. Here we'd like to introduce you to some of the awesome things we've seen out there.
+
+_**Note:** If you miss something that's not listed here and want to see it here, please let us [know](mailto:hello@reactioncommerce.com)._
```

Comment:
Couldn't we just ask people to open a PR rather than yet another way to communicate with us?

Reply:
yup. makes sense